sbpy/activity/gas/tests/test_core.py

```diff
@@ -392,3 +393,60 @@ def test_missing_scipy(self, monkeypatch):
             test._iK0(1)
         with pytest.raises(sbe.RequiredPackageUnavailable):
             test._K1(1)
+
+class TestVectorialModel:
```

Comment:
Could a test be developed that reproduces results from Festou's second paper?  https://ui.adsabs.harvard.edu/abs/1981A%26A....96...52F/abstract

I see some quoted values in Section IV that might help develop a test.

Reply:
If this is too tricky, we can leave it as an open issue.